app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
```

Comment:
- [ ] I think at this point, Rails will be unable to get all the posts written by a user since in the posts table you made author_id the reference key instead of the default post_id Rails will be searching. To resolve this issue, you'll be required to tell Rails that you have changed the default behavior and made author_id the reference key by modifying this line of code to: `has_many :posts, foreign_key: 'author_id'`